Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/model/Version.java

```diff
@@ -74,6 +81,24 @@ public boolean compatibleWith(Version that) {
         return true;
     }
 
+    public boolean matchStrictly(Version that) {
+        return this.version.equals(that.version);
```

Comment:
Maybe a null check to return false if `that` is null?

Reply:
Thanks! Done.